hydra/@context/video.jsonld

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": {
+    "@vocab": "http://videos.restfest.org/",
+    "@base": "http://purl.org/restfest/videos/",
+    "sch": "http://schema.org/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "uri": {
+      "@type": "@id"
+    },
+    "created_time": {
+      "@id": "sch:dateCreated",
+      "@type": "xsd:DateTime"
+    },
+    "modified_time": {
+      "@id": "sch:dateModified",
+      "@type": "xsd:DateTime"
+    },
+    "release_time": {
+      "@id": "sch:datePublished",
+      "@type": "xsd:Date"
+    },
+    "name": "sch:name",
+    "description": "sch:description",
+    "duration": {
+      "@id": "sch:duration",
+      "@type": "sch:Duration"
+    },
+    "width": {
+      "@id": "sch:width",
+      "@type": "sch:QuantitativeValue"
+    },
+    "height": {
+      "@id": "sch:height",
+      "@type": "sch:QuantitativeValue"
+    },
+    "language": {
+      "@id": "sch:inLanguage"
+    },
+    "license": {
+      "@id": "sch:license",
+      "@type": "sch:URL"
+    },
+    "link": {
+      "@type": "@id"
+    }
```

Comment:
Same situation as with `uri` above. This should be `"link": "@id"`. See https://www.w3.org/TR/json-ld/#h3_aliasing-keywords

Reply:
ditto